client/src/api/socketService.js

```diff
@@ -22,9 +22,20 @@ const leaveGame = () => {
   socket.emit('disconnect');
 };
 
+const updateGameState = ({ pacman, roomId, playerId }) => {
```

Comment:
This is not required. There is already a method that does this in a room.

Reply:
I didn't get this point.